python_ta/reporters/templates/template.html

```diff
@@ -23,7 +23,12 @@ <h2> {{ filename }} </h2>
                     <h3 class="category-heading">
                         <span>{{ reporter.code_err_title }}</span>
                         <span class="slider">
-                            <button>Expand/Collapse Section</button>
```

Comment:
It's better for accessibility to keep the button element and just put the svg inside. This will also allow you to remove the inline `cursor: pointer` rule. In general, you shouldn't use inline styles.